packages/labs/ssr/package.json

```
@@ -228,14 +228,16 @@
     "@lit-labs/ssr-dom-shim": "^1.2.0",
     "@lit/reactive-element": "^2.0.4",
     "@parse5/tools": "^0.3.0",
-    "@types/node": "^16.0.0",
     "enhanced-resolve": "^5.10.0",
     "lit": "^3.1.2",
     "lit-element": "^4.0.4",
     "lit-html": "^3.1.2",
     "node-fetch": "^3.2.8",
     "parse5": "^7.1.1"
   },
+  "peerDependencies": {
+    "@types/node": ">=18.0.0 <25.0.0"
+  },
```

For pure JS consumers, we should not impose `@types/node` as a dependency in their project I think.

```suggestion
  },
  "peerDependenciesMeta": {
    "@types/node": {
      "optional": true
    }
  },
```

---

I suggested this, but it was rejected. But I can add it.